packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -837,9 +756,9 @@ add (const Scalar& alpha,
     MM = rcp(new TimeMonitor(*TimeMonitor::getNewTimer(prefix_mmm + std::string("building optimized column map"))));
 #endif
     CrowMap = Bprime->getRowMap();
-    //OLD: CcolMap = AddKern::makeColMapAndConvertGids(Aprime->getGlobalNumCols(), globalColinds, colinds, comm);
-    //Get C's column map as the union of Aprime and Bprime col maps
-    CcolMap = AddKern::makeColMapAndConvertGids(Aprime->getGlobalNumCols(), globalColinds, colinds, comm);
+    colinds = col_inds_array("C colinds", globalColinds.extent(0));
```

Comment:
It looks like this View is an output argument of the function call below, so consider not initializing it here:
```c++
using Kokkos::view_alloc;
using Kokkos::WithoutInitializing;
colinds = col_inds_array(view_alloc("C colinds", WithoutInitializing), globalColinds.extent(0));
```

Reply:
I made the edits locally using the other syntax ``Kokkos::ViewAllocateWithoutInitializing("C colidnds")`` but then I scrolled up and saw this. Is ``view_alloc(..., WithoutInitializing)`` the preferred syntax?